services/logging/pipeline_test.go

```diff
@@ -0,0 +1,159 @@
+package logging
+
+import (
+	"github.com/ionos-cloud/sdk-go-bundle/shared"
+	"github.com/ionos-cloud/sdk-go-bundle/shared/fileconfiguration"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/utils/bundle"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/utils/loadedconfig"
+)
+
+func TestClientConfigurationFlowTable(t *testing.T) {
```

Comment:
These tests are great for what 99% of users will do, but maybe we can add the following in future pr:

- IONOS_CURRENT_PROFILE testing
- IONOS_API_URL overrides IONOS_API_URL_(PRODUCT)
- IONOS_CONFIG_FILE custom location definition (but after looking at the code this might be unnecessary)
- Negative testing , for example, specifying another location in the plan, than the location defined in the config, expect the override to not occur in this case.

Reply:
good points, will add